### DISCUSSION TITLE
Support for Textlocal in Supabase Auth for Phone Login

### DISCUSSION THREAD

**Discussion by Pavan-Rajesh** [supabase/supabase#26538] [Feature Requests]
# Feature Request

## Link 

[Add a link to the page which needs improvement (if relevant)](https://supabase.com/docs/guides/auth/phone-login)

## Describe the problem
Support [Textlocal](https://textlocal.com/)

## Describe the improvement

authentication using textlocal should be provided

## Additional context

Add any other context or screenshots that help clarify your question.

**Comment by encima** [comment]
Hi @Pavan-Rajesh 

Currently, we support [MessageBird](https://supabase.com/docs/guides/auth/phone-login/messagebird), [Twilio](https://supabase.com/docs/guides/auth/phone-login/twilio) and [Vonage](https://supabase.com/docs/guides/auth/phone-login/vonage) for Phone Auth.

I have added this as a feature request for the Auth team. Thanks for the issue!

**Comment by Hallidayo** [comment]
Hi everyone, due to inactivity on this issue I've moved the issue over to discussions (Feature Requests) for future reference
